### ISSUE TITLE
Handling unresolved trees

### ISSUE THREAD

**Issue by annazhukova** [tothuhien/lsd2#14]
Hi Hien,

Could it be possible to add support for trees with politomies in LSD2? Now when I try to date one I get:
```bash
...
Estimating the root position locally around the given root ...
lsd2: malloc.c:2379: sysmalloc: Assertion `(old_top == initial_top (av) && old_size == 0) || ((unsigned long) (old_size) >= MINSIZE && prev_inuse (old_top) && ((unsigned long) old_end & (pagesize - 1)) == 0)' failed.
```
If I randomly resolve politomies by adding zero branches, everything works.

Or at least a more informative error message should be added, so the user knows what to do :)

Thanks,
Anna

**Comment by tothuhien** [comment]
Hi Anna,
There was a bug in my commit c13a2f2. I've fixed that in the latest one. Not sure if it's related to this issue. Could you try again and there's still problem? I haven't tested on unresolved trees with supports, but it supposes to be able to deal with it. I'll try to find the issue, if you have an example for me to debug, that'd be very helpful, thanks!
Hien
Hien

**Comment by tothuhien** [comment]
The problem is resolved in the commit 405fa8a
